mne_bids/path.py

```diff
@@ -599,14 +599,6 @@ def fpath(self):
 
                 # found no matching paths
                 if not matching_paths:
-                    msg = (f'Could not locate a data file of a supported '
-                           f'format. This is likely a problem with your '
```

Comment:
This warning is unnecessary as we don't intend to guarantee that the path exists, and it sometimes showed up in weird situations unexpectedly, causing confusion.

To check for existence of the path, users should call `BIDSPath.fpath.exists()`. This is mentioned in the changelog.